### PR TITLE
to_crs doctring proj4 instructions

### PR DESCRIPTION
**Pull request by rutgerhofste**

### CODE REVIEW

File: geopandas/geodataframe.py [hunk 1]
```diff
@@ -417,18 +417,23 @@ def to_crs(self, crs=None, epsg=None, inplace=False):
         Transform all geometries in a GeoSeries to a different coordinate
         reference system.  The ``crs`` attribute on the current GeoSeries must
         be set.  Either ``crs`` in string or dictionary form or an EPSG code
-        may be specified for output.
+        may be specified for output. crs proj4 strings can be found at 
+        https://epsg.io/ . Make sure to click proj4 instead of copying the
+        definition string on top of the page.
```
[the remark this answers is not in the document]
Comment (rutgerhofste) [reply geopandas/geodataframe.py]: For example: http://epsg.io/54012 The difference is the +ellps=WGS84 that causes an error:
http://nbviewer.jupyter.org/gist/rutgerhofste/b174e4c8fa4c9a2ad40e60bb2e482c0c

[the remark this answers is not in the document]
Comment (jorisvandenbossche) [reply geopandas/geodataframe.py]: Ah, OK, I also saw your comment on StackOverflow in the meantime.

**Comment by valgur** [comment]
A minor nitpick: the PROJ.4 project was renamed to just PROJ earlier this year with the release of version 5. https://proj4.org/news.html#proj-5-0-0

So you might want to refer to it without the number.

**Comment by snowman2** [comment]
Just so you are aware, `pyproj` 2+ has been update to accept many different types of input.

- PROJ string
- Dictionary of PROJ parameters
- PROJ keyword arguments for parameters
- JSON string with PROJ parameters
- CRS WKT string
- An authority string [i.e. ‘epsg:4326’]
 - An EPSG integer code [i.e. 4326]
- A tuple of (“auth_name”: “auth_code”) [i.e (‘epsg’, ‘4326’)]
- An object with a to_wkt method.
- PROJ JSON
- OGC urn

**Comment by martinfleis** [comment]
As per @snowman2's comment, this PR seems to be obsolete thanks to the updates of pyproj. We aim to implement pyproj crs class, so we will rework relevant section of docs anyway. Thank you @rutgerhofste for this PR anyway.

**Pull request closed without merging**